website/static/js/tests/paginator.test.js

```diff
@@ -0,0 +1,148 @@
+/*global describe, it, expect, example, before, after, beforeEach, afterEach, mocha, sinon*/
+'use strict';
+
+var Paginator = require('js/paginator');
+var oop = require('js/oop');
+var assert = require('chai').assert;
+sinon.assert.expose(assert, {prefix: ''});
+
+var spy = new sinon.spy();
+
+var TestPaginator = oop.extend(Paginator, {
+    constructor: function(){
+        this.super.constructor();
+    },
+    search: spy,
+    configure: function(config){
+        config(this);
+    }
+});
+
+
+describe('Paginator', () => {
+    var paginator;
+    var numberOfPages;
+    var currentPage;
+
+    beforeEach(() => {
+        paginator = new TestPaginator();
+    });
+
+    afterEach(() => {
+        spy.reset();
+        numberOfPages = 0;
+        currentPage = 0;
+    });
+
+    it('previousPage', () => {
+        numberOfPages = 5;
+        currentPage = 3;
+        paginator.configure(function(p){
+            p.numberOfPages(numberOfPages);
+            p.currentPage(currentPage);
+        });
+        paginator.previousPage();
+        assert.isTrue(paginator.search.calledOnce);
+        assert.equal(paginator.currentPage() + 1, currentPage);
+    });
+
+    it('nextPage', () => {
+        numberOfPages = 5;
+        paginator.configure(function(p){
+            p.numberOfPages(numberOfPages);
+        });
+        paginator.nextPage();
+        assert.isTrue(paginator.search.calledOnce);
```

Comment:
You should be able to do `assert.calledOnce(paginator.search)` since we expose sinon asserts on the `assert` object.